Sources_v3/WebSocketClient/ChatClientConnectionStatus.swift

```diff
@@ -4,12 +4,49 @@
 
 import Foundation
 
-public typealias ConnectionId = String
+// `ChatClientConnectionStatus` is just a simplified and friendlier wrapper around `WebSocketConnectionState`.
+
+/// Describes the possible states of the client connection to the servers.
+public enum ChatClientConnectionStatus: Equatable {
+    /// The client is disconnected. This is an initial state. Optionally contains an error, if the connection was disconnected
+    /// due to an error.
+    case disconnected(error: ClientError? = nil)
```

Comment:
I liked how it was in `ConnectionState` as `notConnected`, but if you decide to change it, I think it should be changed in `ConnectionState` too to avoid confusing.

Reply:
Renamed to `disconnected` in a219d59